builtin/plugins/chainconfig/chainconfig.go

```diff
@@ -186,6 +186,27 @@ func (c *ChainConfig) AddFeature(ctx contract.Context, req *AddFeatureRequest) e
 	return nil
 }
 
+//Adds specific feature with name provided
+func AddSpecificFeature(ctx contract.Context, name string, blockHeight uint64) error {
```

Comment:
Name should be along the line of `SyncEnabledFeature`.

Also, in case feature already exists, toggle its status to `Enabled`, otherwise create a new feature and store it.

Reply:
Have one question, is it necessary migration will always enable a feature? it might enable a feature, disable another feature,  so depending on what was done at the time of migration, we need to preserve that state right ..
So in case feature already exists, it might have enabled/disabled state

Reply:
We are querying enabled features from state only.

Reply:
Done

Reply:
One question, though we are querying for enabled features , should not we store all the features enabled/disabled at the time of migration in chainconfig context, so that state and context are in sync.